docs/subnets/how-to-transfer-funds.md

```diff
@@ -0,0 +1,332 @@
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+| ledger | index 0 | P-Chain (Bech32 format) | P-avax1r4aceznjkz8ch4pmpqrmkq4f3sl952mdrdt6xm |       0 | Mainnet |
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+```
+
+Target address to be used is `P-avax1r4aceznjkz8ch4pmpqrmkq4f3sl952mdrdt6xm`, containing 0 funds.
+
+### Execute the Sending Part of the Transfer Operation
+
+A P-Chain to P-chain transfer is a two-part operation. There is no need for the two parts to be executed
+on the same machine, only for them to have some common params. For each part, the appropriate ledger
+(either source or target) must be connected to the machine executing it.
+
+First part starts the process, and moves the money out of the
+source account into a X-Chain account owner by the receptor. It needs to be signed by the sending ledger.
+
+The amount to be specified as input to the command is the amount to be received on the target address.
+
+As in this example the users wants to send all the balance of 4.5, so he needs to consider a fee payment
+of 0.004, and tell the command to transfer 4.496.
+
+Take into account that the sending ledger is the one that is going to pay all the fees.
+
+Then start the command:
+
+```bash
+avalanche key transfer
+```
+
+First step is to specify the network. `Mainnet` in this case:
+
+```text
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Network to use:
+  ▸ Mainnet
+    Fuji
+    Local Network
+```
+
+Next, the step of the transfer must be specified. Send in this case:
+
+```text
+? Step of the transfer:
+  ▸ Send
+    Receive
+```
+
+Next, the key source for the sender address. That is, the key that is going to sign the sending
+transactions. Select `Use ledger`:
+
+```text
+? Which key source should be used to  for the sender address?:
+    Use stored key
+  ▸ Use ledger
+```
+
+Next, the ledger index is asked for. Input `2`:
+
+```text
+✗ Ledger index to use: 2
+```
+
+Next, the amount to be sent is asked for:
+
+```text
+✗ Amount to send (AVAX units): 4.496
+```
+
+The, the target address is required:
+
+```text
+✗ Receiver address: P-avax1r4aceznjkz8ch4pmpqrmkq4f3sl952mdrdt6xm
+```
+
+After that, a confirmation message is printed. Read carefully and choose `Yes`:
+
+```text
+this operation is going to:
+- send 4.496000000 AVAX from P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0 to target address P-avax1r4aceznjkz8ch4pmpqrmkq4f3sl952mdrdt6xm
+- take a fee of 0.004000000 AVAX from source address P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0
+
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Confirm transfer:
+    No
+  ▸ Yes
+```
+
+After this, the first part is completed:
+
+```text
+Issuing ExportTx P -> X
+```
+
+### Execute the Receiving Part of the Transfer Operation
+
+This part of the operation, to be signed by ledger B, is going to take the funds from a X account owned
```

Comment:
In this step, Ledger B signs the transaction to receive the funds. It imports the funds on the X-Chain before exporting them back to the desired P-Chain address.